Sources/_CryptoExtras/AES/AES_CBC.swift

```diff
@@ -18,7 +18,7 @@ import Foundation
 extension AES {
     /// The Advanced Encryption Standard (AES) Cipher Block Chaining (CBC) cipher
     /// suite.
-    public enum _CBC {
+    public enum _CBC: Sendable {
```

Comment:
This enum is uninhabited, it doesn't need a `Sendable` annotation.

Reply:
Ah, yep, good catch, I missed that on this pass.